recipes/cassandra-driver/meta.yaml

```diff
@@ -0,0 +1,58 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "cassandra-driver" %}
+{% set version = "3.8.1" %}
+{% set sha256 = "" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
+    - setuptools
+    - six >=1.6
+    - futures <=2.2.0
+    - cython <0.25
+
+  run:
+    - python
+    - numpy
+    - six >=1.6
+    - futures <=2.2.0
```

Comment:
Same here. Add a `# [py2k]` selector to the `futures` requirement.  e.g., `- futures <=2.2.0  # [py2k]`